site/public/js/server.js

```diff
@@ -115,6 +135,106 @@ function newClassListForm() {
     $('[name="upload"]', form).val(null);
 }
 
+function copyStudentEmail(code,emails,emails_corresponding_section) {
```

Comment:
following two functions are js functions which actually performs copy or download when button on form is clicked.
1st function is for student tab and next for grader tab for instructor user